### DISCUSSION TITLE
Can you please update the mod to 1.19?

### DISCUSSION THREAD

**Discussion by renja-g** [LXYan2333/Fabric-Bedrock-Miner#39] [Ideas]
Can you please update the mod to 1.19?

**Comment by loctran016** [comment]
There is a [1.19 version of this mod](https://github.com/LXYan2333/Fabric-Bedrock-Miner/suites/6957582944/artifacts/271525089) shown by LXYan2333 in issue #35

**Reply by IktsoisAlt** [reply]
I wish links would work.

**Comment by df-f** [comment]
I placed both the files extracted from Artifact.zip into mod folder, but the game isn't loading... any tip?

**Reply by LXYan2333** [reply]
delete the `bedrock-miner-1.0.0-sources.jar`

**Reply by df-f** [reply]
thanks, i made it! thanks again for this mod!

**Reply by tasoisbadatcoding** [reply]
Could you send me the download? the page is gone:(

**Reply by Yeah4520** [reply]
Could someone please send me the download its gone :(

**Comment by LXYan2333** [comment]
1.19.2 and 1.19.4 avaliable at https://modrinth.com/mod/fabric-bedrock-miner
